packages/react/src/components/Cta/__stories__/Cta.stories.js

```diff
@@ -0,0 +1,118 @@
+import { select, withKnobs } from '@storybook/addon-knobs';
+import CTA from '../CTA';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Components|CTA', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    let cta, type;
+    const copy = ['Lorem ipsum dolor sit amet', 'Consectetur adipisicing elit'];
+    const types = ['local', 'jump', 'external'];
+    const styles = ['text', 'card', 'feature', 'button'];
+    const style = select('style', styles, styles[0]);
+
+    const urlBy = {
+      jump: '#example',
+      local: 'https://ibm.com',
+      external: 'https://www.example.com',
+    };
+
+    switch (style) {
+      case 'text':
+        type = select('type', types, types[0]);
+        cta = {
+          type: type,
+          href: urlBy[type],
+          copy: copy[0],
+        };
+        break;
+      case 'card':
+        type = select('type', types, types[0]);
+        cta = {
+          type: type,
+          href: urlBy[type],
+          title: copy[0],
+        };
+        break;
+      case 'feature':
+        type = select('type', types, types[0]);
+        cta = {
+          heading: copy[0],
+          card: {
+            type: type,
+            href: urlBy[type],
+            title: copy[1],
+            image: {
+              defaultImage: 'https://picsum.photos/id/672/672',
+              alt: '...',
```

Comment:
Might be good to add an actual alt text here. `Placeholder image`.

Reply:
done